build.gradle

```diff
@@ -28,7 +28,8 @@ buildscript {
 }
 
 plugins {
-  id "com.palantir.docker" version "0.20.1"
```

Comment:
should we standardize on single quotes?

Reply:
Quoting in gradle is similar to bash: double quotes allow interpolation; single quotes do not.
For example, if we had something like:
```
def palantir_version = '0.20.1'
plugins {
  id "com.palantir.docker" version "$palantir_version"
}
```

That said, I'm happy to use single quotes as the default unless there is a reason to use doubles.

Reply:
Added a section in our SDLC notes about code standards: https://docs.google.com/document/d/1Dx4Zy_IcOzwb_2d4S7pP56MnLD_r6jpkAq9dsWpT0GA/edit#

Reply:
Separately, this plugin needs to be enabled for dockerization to work. I'll update